leaderboard/models.py

```diff
@@ -82,8 +83,6 @@ def validate_sgml_schema(hyp_file):
 
     try:
         schema.validate(hyp_file)
-
-    # pylint: disable-msg=bad-continuation
```

Comment:
@snukky why are you dropping the `# pylint: disable-*` directives?

Reply:
Because these particular directives were needed only because pylint didn't know about Django objects, which has been fixed by adding the pylint-django plugin in https://github.com/AppraiseDev/OCELoT/commit/dc3ca3b478ae33dbe5c062f1b422d60d662da4f9#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52